app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -51,6 +59,7 @@ def perform(date = Time.zone.today)
           tables: email_tables,
         ).deliver_now
       else
+        # MW -- pull this up sooner, don't run if this is the case.
```

Comment:
Oh, FIXME: This should be its own pull request. _But_, I do think we should check this at the start rather than generating the report and potentially throwing it away... Just not in this PR.